fdbrpc/Locality.h

```diff
@@ -180,6 +180,8 @@ struct LocalityData {
 	Optional<Standalone<StringRef>> dcId() const { return get(keyDcId); }
 	Optional<Standalone<StringRef>> dataHallId() const { return get(keyDataHallId); }
 
+	bool isValidLocalityValueUnderPolicy(std::string policy) { return get(policy).present(); }
```

Comment:
Looking at this closer, it appears there exists a function to check this already: `isPresent(key)`. We should probably just use that instead.

Reply:
Thanks!